src/main/java/io/gravitee/resource/oauth2/keycloak/OAuth2KeycloakResource.java

```diff
@@ -106,6 +108,14 @@ protected void doStart() throws Exception {
                 .setDefaultPort(authorizationServerPort)
                 .setDefaultHost(authorizationServerHost);
 
+        //Use proxy if it is set in properties
+        if (System.getProperty("https.proxyHost") != null && System.getProperty("https.proxyPort") != null){
```

Comment:
Wouldn't it be preferable to be able to configure HTTP proxy from the Keycloak resource instead of using system properties ?